content/blog/introducing-python-sdk-for-dscc.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Introducing Python SDK for DSCC
+date: 2023-03-29T13:45:50.979Z
+priority: 0
+author: Anusha Y and Sijeesh Kattumunda
+authorimage: /img/Avatar1.svg
+disable: false
+---
+In my previous blog, I discussed how to generate a Software Development Kit (SDK) from the Data services cloud console (DSCC) Open API spec using a third-party tool called OpenAPI generator. Today, I am going to talk more in detail on the Python SDK generated using the tool. As we progress, we will discuss the implementation aspects, and then take up a use-case to better understand how to use the SDK. 
+Before we get into SDK, let’s understand the difference between an Application Programming Interface (API) and an SDK. 
```

Comment:
Make sure to use some line breaks to separate the paragraphs in the introduction section, similar to what you have in the MS-Word document.